core/src/main/java/nl/nn/adapterframework/webcontrol/api/ServerStatistics.java

```diff
@@ -312,8 +312,9 @@ public Response updateLogConfiguration(LinkedHashMap<String, Object> json) throw
 				Level loglevel = Level.toLevel(""+value, null);
 				Logger rootLogger = LogUtil.getRootLogger();
 				if(loglevel != null && rootLogger.getLevel() != loglevel) {
+					String changmsg = "LogLevel changed from [" + rootLogger.getLevel() + "] to [" + loglevel +"]";
 					Configurator.setLevel(rootLogger.getName(), loglevel);
-					msg.append("LogLevel changed from [" + rootLogger.getLevel() + "] to [" + loglevel +"]");
+					msg.append(changmsg);
```

Comment:
?

Reply:
In de oude situatie was de 'from' al veranderd.
Er stond dan: LogLevel changed from [DEBUG] to [DEBUG]